allensdk/brain_observatory/behavior/session_apis/data_io/behavior_ophys_nwb_api.py

```diff
@@ -127,6 +133,13 @@ def save(self, session_object):
         # Add motion correction to NWB in-memory object:
         nwb.add_motion_correction(nwbfile, session_object.motion_correction)
 
+        # Add eye tracking to NWB in-memory object:
+        self._add_eye_tracking_data_to_nwb(nwbfile=nwbfile,
```

Comment:
I think this should actually look more like:

```
nwb.add_eye_tracking(nwbfile, session_object.get_eye_tracking)
```

Where the nwb namespace is from:
https://github.com/AllenInstitute/AllenSDK/blob/a337da9fbda272b738226372afc110c46057e206/allensdk/brain_observatory/nwb/__init__.py

Reply:
Note that this is a private method which calls individual `add_*` methods for each component of eye tracking data. Eye tracking is not 1 piece of data but multiple. Do you still have the same comment?

Reply:
ah, got it. That should be fine then.